release-notes.md

```diff
@@ -1,5 +1,70 @@
 # Release Notes
 
+## 3.0
+
+medic-conf v3.0 contains breaking changes! This release only impacts the `compile-app-settings` action and the partner code written in `tasks.js`, `targets.js`, `contact-summary.js`, and `contact-summary.templated.js`.
+
+### Breaking Change - Static Analysis via EsLint
+medic-conf has changed the engine we use for performing static analysis from [JsHint](https://jshint.com/) to [eslint](https://eslint.org/). This grants authors more control over the static analysis rules which are enforced on their code. We provide a set of default rules, but you can create an [.eslintrc file](https://eslint.org/docs/user-guide/configuring#using-configuration-files-1) to define your override these defaults or define your own static analysis rules. The default static analysis rules which are enforced in medic-conf 3.0 are similar to the rules which were enforced in medic-conf 2.x, but details may be slightly different.
```

Comment:
> to define your override these defaults or define your own static analysis rules.

Should perhaps be: "to define **or** override these defaults or define your own static analysis rules."

Reply:
Went with `to override these defaults or add rules`